examples/distro/README.md

```diff
@@ -24,3 +24,36 @@ customize that
 * [DemoSpanProcessor](custom/src/main/java/com/example/javaagent/DemoSpanProcessor.java) - custom `SpanProcessor`
 * [DemoSpanExporter](custom/src/main/java/com/example/javaagent/DemoSpanExporter.java) - custom `SpanExporter`
 * [DemoServlet3Instrumentation](instrumentation/servlet-3/src/main/java/com/example/javaagent/instrumentation/DemoServlet3Instrumentation.java) - additional instrumentation
+
+## Instrumentation customisation
```

Comment:
British spelling, eh? Should we standardise on the British, or American spellings i wonder?

Reply:
I am simple Russian guy :). Whatever spelling suggests me in the specific tool...